### ISSUE TITLE
Does not install and register vgpusrv service

### ISSUE THREAD

**Issue by kevinoid** [virtio-win/virtio-win-guest-tools-installer#32]
Based on https://bugzilla.redhat.com/show_bug.cgi?id=1923886#c4, it's my understanding that `viogpuap.exe` and `vgpusrv.exe` should be installed and `vgpusrv.exe -i` run for service registration to provide resolution switching for the viogpudo driver.  However, it does not appear that the installer does this when viogpudo is selected.  Is it something that could/should be added?

Thanks,
Kevin

**Comment by vrozenfe** [comment]
@kevinoid 
Right, we do not install and start the automatically automatically.
The reason for that is quite simple. It is still not clear how the dynamic
resolution should be integrated with VNC and SPICE vda agent.

As soon as we know the requirements, It shouldn't be too complicated 
to include these binaries  in installer.

Best,
Vadim.

**Comment by vrozenfe** [comment]
Unfortunately, there is no progress on this issue yet. 
Regards,
Vadim.

**Comment by matwey** [comment]
For me, `vgpusrv.exe` also currently doesn't work with SPICE display, but works well with VNC.

**Comment by vrozenfe** [comment]
Unfortunately SPICE is not supported by Red Hat any longer.
And as far as I know there is no plan to integrate viogpudo with SPICE.
vgpusrv.exe supposed to work with VNC but not with SPICE because 
SPICE uses it's own communication mechanism to notify qxl in case of 
resolution change event while  virtio-gpu device uses a dedicated interrupt 
for that.

Best,
Vadim.

**Comment by karypid** [comment]
I am trying to get auto-resize to work with my VM and it only works with QXL.

I want to use VirtIO instead but when I switch to that auto-resize stops working.

What is the "best" supported configuration? Is it VirtIO with VNC only?

**Comment by tuxmaster5000** [comment]
Here SPICE and VirtIO works fine using resizing. Guest OS: Win-10, Host: Fedora-42. But the manual install of the service are annoying. Using:
`remote-viewer spice://<HOST>:<PORT>`

**Comment by kevinoid** [comment]
It appears that the [virtio-win-gt-x64.msi version 0.1.302-1](https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/archive-virtio/virtio-win-0.1.302-1/virtio-win-gt-x64.msi) now provides a "VioGpu Resolution Service" component (which is disabled by default) as a result of 4899fa7c81de736f233063a1df4341d0bc46a4fb.  This fits my needs.  Thanks @xz-dev and @YanVugenfirer!

Can this issue be closed, or does it make sense to leave it open until the integration with VNC and SPICE vda agent is sorted out more fully (or the service is enabled by default)?

**Comment by YanVugenfirer** [comment]
Hi @kevinoid , I think the issue can be closed.

**Comment by candrews** [comment]
Can we keep this open until the service is enabled by default?
